lib/octokit/connection.rb

```diff
@@ -160,7 +160,7 @@ def request(method, path, data, options = {})
     # @return [Boolean] True on success, false otherwise
     def boolean_from_response(method, path, options = {})
       request(method, path, options)
-      @last_response.status == 204
+      @last_response.status == 204 || @last_response.status == 201
```

Comment:
I see what you mean. Do you think it would be better to return the response here even if it's empty now instead of a Boolean?